rasa/utils/tf_models.py

```diff
@@ -44,42 +46,51 @@ def fit(
                 f"Validation accuracy is calculated every {evaluate_every_num_epochs} "
                 f"epochs."
             )
+
+            model_data, evaluation_model_data = model_data.split(
+                evaluate_on_num_examples, random_seed, label_key=label_key
+            )
+
         disable = silent or is_logging_disabled()
         pbar = tqdm(range(epochs), desc="Epochs", disable=disable)
 
         tf_batch_size = tf.ones((), tf.int32)
 
         def train_dataset_function(x):
```

Comment:
```suggestion
        def train_dataset_function(_batch_size):
```
to remember what the argument is